pkgs/tools/misc/empty/default.nix

```diff
@@ -0,0 +1,44 @@
+{ fetchzip, lib, stdenv, which }:
+
+stdenv.mkDerivation rec {
+  pname = "empty";
+  version = "0.6.21b";
+
+  src = fetchzip {
+    url = "mirror://sourceforge/${pname}/${pname}/${pname}-${version}.tgz";
+    sha256 = "1rkixh2byr70pdxrwr4lj1ckh191rjny1m5xbjsa7nqw1fw6c2xs";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [ which ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  # Allow installation prefix to be overridden, and fix the CC parameter order.
+  prePatch = ''
+    substituteInPlace Makefile \
+      --replace '\$\{LIBS\} -o empty empty.c' '-o empty empty.c \$\{LIBS\}'
```

Comment:
> empty> substituteStream(): WARNING: pattern '\$\{LIBS\} -o empty empty.c' doesn't match anything in file 'Makefile'
```suggestion
      --replace '${LIBS} -o empty empty.c' '-o empty empty.c ${LIBS}'
```

Reply:
Gotcha - thanks for the pull request - it's been merged!